data-access/basic_data_access.ipynb

```diff
@@ -167,538 +248,1117 @@
+    "        },\n",
+    "        \"clip\": location,\n",
+    "        \"convert_units\": \"degF\",\n",
+    "        \"export\": {\n",
+    "            \"filename\": \"gridded_export_data\", # Filename that you want the data exported to\n",
+    "        }\n",
+    "    })\n",
+    "    .get()\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1cc9720a-fd39-4830-ad10-78e6fe48e4fe",
    "metadata": {},
+   "outputs": [],
    "source": [
-    "Zarr format is technically a directory, not a single file, because it uses chunking to write and read data. This cloud-optimized format enables some unique benefits for performing data computations in a cloud computing environment like the AE Jupyter Hub, but can also make it tricky to delete. Because of that, we have built a simple helper function to facilitate easily deleting zarrs. "
+    "lat_lons = [\n",
+    "    (34.05, -118.25),  # Los Angeles, CA\n",
+    "    (37.77, -122.42),  # San Francisco, CA\n",
+    "]"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "a28e241e",
+   "id": "b7e81b62-1904-4995-997d-0e9238ccb8ea",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# ck.remove_zarr(\"my_filename2\") # Helper function to delete Zarr directory tree"
+    "# Now, let's export this list of `lat_lons` from before into separate files.\n",
+    "export_data = (\n",
+    "    cd.catalog(\"cadcat\")\n",
+    "    .activity_id(\"WRF\")\n",
+    "    .institution_id(\"UCLA\")\n",
+    "    .table_id(\"mon\")\n",
+    "    .grid_label(\"d03\")\n",
+    "    .variable(\"t2\")\n",
+    "    .processes({\n",
+    "        \"warming_level\": {\n",
+    "            \"warming_levels\": [1.2],\n",
+    "        },\n",
+    "        \"clip\": {\n",
+    "            \"points\": lat_lons,\n",
+    "            \"separated\": True,\n",
+    "        },\n",
+    "        \"convert_units\": \"degF\",\n",
+    "        \"export\": {\n",
+    "            \"separated\": True, # (ONLY FOR POINT-BASED CLIPPING) Whether or not you want different points to be separated into different files\n",
+    "            \"location_based_naming\": True, # Whether or not you want the lat/lons of different points passed \n",
+    "            \"filename\": \"point_export_data\", # Filename that you want the data exported to\n",
+    "        }\n",
+    "    })\n",
+    "    .get()\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a5c04a36",
+   "metadata": {},
+   "source": [
+    "We hope this notebook and `climakitae` help empower you in your future climate data analyses!\n",
+    "\n",
+    "**If you have any issues running this notebook, or are running into any issues with any of these processors, or have general feedback about the behavior of the `ClimateData` object or any of its processors, please write an Issue on our Github repo here: https://github.com/cal-adapt/climakitae/issues. Thank you so much!**"
```

Comment:
I'd like to see this final section be less of a run-on; it feels a little too casual.

Reply:
Agreed. Maybe give a recap of some of the highlights, encourage users to try it out, and then direct them to the github for issues or new features.